src/scss/components/_sidebar.scss

```diff
@@ -6,6 +6,32 @@
 }
 
 .sidebar__header {
+  position: static;
+  top: 0;
+  z-index: 1;
+  background-color: $react-timelines-sidebar-background-color;
+}
+
+.sidebar__header {
+  position: relative;
+  margin-left: ($react-timelines-sidebar-width - $react-timelines-sidebar-closed-offset);
+  direction: rtl;
```

Comment:
Onus is on the `.is-sticky` functionality to set this stuff if required. Imagine stripping out `.is-sticky` - don't want residual changes left behind.